### PR TITLE
physics/mechanics Documentations and Examples for Joints API

### PR DESCRIPTION
**Pull request by sahilshekhawat**

### CODE REVIEW

**Comment by sahilshekhawat** [comment]
@moorepants Is it the correct way of describing things? Just talking about general things rather than the working.

**Comment by moorepants** [comment]
Yeah, that's a good start. You want to imagine that you are a user of your new classes and think about what you'd want to know to get started with them. You'd probably just want to start with something like "The joints method allows you to construct serial and tree chains of rigid bodies with different types of joints, much like you would do if you were actually constructing such a system with physical bodies." And then go one telling the "story" of what this is all about and what the pieces are. This can include simple examples throughout the text, etc. Finally, tutorials are usually very helpful, i..e show someone how to construct a system of some sort. Like how to build a double pendulum or a robot arm, etc.

File: doc/src/modules/physics/mechanics/joints.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+====================================================
+Body, Joints and Joint's Method in Physics/Mechanics
+====================================================
+
+:mod:`mechanics` provides functionality for deriving equations of motion
+using Kane's method [Kane1985]_. This document will describe Kane's method
+as used in this module, but not how the equations are actually derived.
```
Comment (moorepants) [inline doc/src/modules/physics/mechanics/joints.rst]: ? Why are you explaining Kanes Method?

**Comment by moorepants** [comment]
This isn't even close to complete. Please ping us when you have something to review.

**Comment by czgdp1807** [comment]
@moorepants Can we close this? Or is there a possibility of continuing it?

**Pull request closed without merging**